x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/degraded_fields/spark_plot.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLoadingChart,
+  euiPaletteColorBlind,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+import { ScaleType, Settings, Tooltip, Chart, BarSeries } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+import { Coordinate } from '../../../../common/types';
+
+export function SparkPlot({
+  valueLabel,
+  isLoading,
+  series,
+}: {
+  valueLabel: React.ReactNode;
+  isLoading: boolean;
+  series?: Coordinate[] | null;
+}) {
+  return (
+    <EuiFlexGroup
+      justifyContent="flexStart"
+      gutterSize="xs"
```

Comment:
```suggestion
      gutterSize="s"
```

The design has `8px` gap.